src/main/resources/application.properties

```diff
@@ -1,38 +1,37 @@
 #Faclity FHIR server
-#org.openelisglobal.facilitylist.fhirstore = https://hub.openelisci.org:5000/gofr/exemple4
-#org.openelisglobal.facilitylist.authurl= https://hub.openelisci.org:5000/gofr/token
-#org.openelisglobal.facilitylist.username= root@gofr.org
-#org.openelisglobal.facilitylist.password= gofr
-#org.openelisglobal.facilitylist.auth= token
+org.openelisglobal.facilitylist.fhirstore = https://hub.openelisci.org:5000/gofr/exemple4
```

Comment:
Disacrd the changes in this file . 
we dont need all the properties by defailt

Reply:
you can just add them to the [common properties file](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/volume/properties/common.properties)  but wih empty values to avoind setting values we dont need by default

Add to the common properries file those which dockt already exist there.
And you can uncomment [these](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/volume/properties/common.properties#L20-L24)  but leave them with empty values

Reply:
@konavivekramakrishna , i already gave you a list of prorties we need but i like the  idea of just reading them directly reading from the spring configuration as opposed to hardcoding them again in the cotroller. 

But the challenge is that it will read all proprties including the sensitive ones. 
I think its just easier to create a List of those we want to ignore as opposed to hardcoding all the properties